dimod/binary/binary_quadratic_model.py

```diff
@@ -758,6 +758,93 @@ def add_linear_equality_constraint(
                     u, v, 2 * lagrange_multiplier * ubias * vbias)
         self.offset += lagrange_multiplier * constant * constant
 
+    def add_linear_inequality_constraint(
+            self, terms: Iterable[Tuple[Variable, Bias]],
+            lagrange_multiplier: Bias,
+            label: str,
```

Comment:
this label is not kept anywhere so I don't think it should be an input.

Reply:
The label goes to slack_terms and slack_terms are returned.

Reply:
oops, missed that! thanks